crates/wasi-crypto/wasmtime/Cargo.lock

```diff
@@ -0,0 +1,1863 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
```

Comment:
Hm so ideally we only have one `Cargo.lock` in the root of the repository, but if one is showing up here is the workspace perehaps misconfigured?

Reply:
Removed, thanks!